OneCommunityGlobal/HighestGoodNetworkApp
# Yubo fix manage 'adminLinks in user profile' permission

# Description
<img width="759" alt="Frontend_PR_Description" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/57503876/88f29a91-d677-4b53-86ae-c57557c08754">


Fixes # 4 in PERMISSIONS MANAGEMENT COMPONENT (medium)

## Related PRS (if any):
It's related to PR #1001, which hasn't been merged into dev yet.
…

## Main changes explained:
- Fix the `canEdit` permission in `UserLinkLayout.jsx` by importing `hasPermission()`.
- Remove some unused props from parent component `UserLinkLayout` to its children components.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as **Owner** user first (it's higher than **admin**, thus you need to ask to upgrade your dev account to **Owner**)
4. go to dashboard→ Other Links→ Permissions Management→ Manage User Permissions (a button on the right side) -> choose a user (let's call it **User B**) you want to grant the permission (need to search the user in the input area) -> choose **Manage Admin Links in User Profile** and submit
5. log as **User B** choose any other users to go to their **User Profile** page -> verify you can edit the Links under their Picture.
6. click `update` function so, but notice you can't **save** as so far you don't have that permission.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/57503876/bdbbb8fb-e3c4-40b4-818b-f46e8e77bcc5


## Note:
# !!At this moment, you can't **Save**!
